setup.cfg

```diff
@@ -1,9 +1,143 @@
+[metadata]
+name = evalml
+version = attr: evalml.__version__
+author = Alteryx, Inc.
+license = BSD 3-clause
+license_files = 
+    LICENSE
+long_description = file: READEME.md
+url = https://github.com/alteryx/evalml/
+download_url = https://pypi.org/project/evalml/
+long_description_content_type = text/markdown
+author_email = open_source_support@alteryx.com
+keywords = data science machine learning optimization automl
+description = EvalML is an AutoML library that builds, optimizes, and evaluates machine learning pipelines using domain-specific objective functions.
+
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    Topic :: Software Development
+    Topic :: Scientific/Engineering
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
```

Comment:
We support 3.9 and 3.10 in testing, so I would delete 3.7 and 3.10 but keep `:: 3`